src/main/java/com/google/devtools/build/lib/packages/BazelStarlarkContext.java

```diff
@@ -68,6 +68,8 @@ public class BazelStarlarkContext implements StarlarkThread.UncheckedExceptionCo
   public enum Phase {
     WORKSPACE,
     LOADING,
+    /** Evaluation for a rule initializer, which does not allow all loading phase operations. */
+    INITIALIZER,
```

Comment:
Introducing a new phase was avoided before and introducing it now for a single function looks like an overkill.

This also potentially interacts with other locations that use phases. And potentially also with symbolic macros.

@brandjon